sim/druid/ferocious_bite.go

```diff
@@ -60,7 +60,7 @@ func (druid *Druid) registerFerociousBiteSpell() {
 			result := spell.CalcAndDealDamage(sim, target, baseDamage, spell.OutcomeMeleeSpecialHitAndCrit)
 
 			if result.Landed() {
-				druid.SpendEnergy(sim, excessEnergy, spell.Cost.(*core.EnergyCost).ResourceMetrics)
+				druid.SpendEnergy(sim, excessEnergy, spell.Cost.SpellCostFunctions.(*core.EnergyCost).ResourceMetrics)
 				druid.ApplyFeral4pT12(sim)
 				druid.SpendComboPoints(sim, spell.ComboPointMetrics())
```

Comment:
Is there an analogous helper to `spell.ComboPointMetrics()` for `spell.EnergyMetrics()` that could be called here?